### PR TITLE
report: center and style top-level runtimeError

### LINKED ISSUE #7011: Consistent style for lh-warnings
Mentioned in #6989.  I think the lh-warnings style looks out of place with the report style e.g.
image

My suggestion is to replace the 1px border with a light orange background (#fff5e3):
image

Would love some thoughts. @hwikyounglee

### PR DESCRIPTION
**Pull request by connorjclark**
Fixes #7011

Before:
See #9270

After: 
image
image

### CODE REVIEW

File: lighthouse-core/report/html/report-styles.css [hunk 1]
```diff
@@ -993,8 +993,9 @@
 .lh-warnings--toplevel {
   --item-margin: calc(var(--header-line-height) / 4);
   color: var(--report-text-color-secondary);
-  margin: var(--category-padding);
-  padding: var(--category-padding);
+  margin-left: auto;
+  margin-right: auto;
+  width: 50%;
```
Comment (patrickhulce) [inline lighthouse-core/report/html/report-styles.css]: is this really what we want? feels like we should be having some sort of viewport width breakpoints to make it full size at smaller screen sizes, etc

**Comment by brendankenny** [comment]
also, can this fix up for #7011?

File: lighthouse-core/report/html/report-styles.css (hunk 1 above)
Comment (connorjclark) [reply lighthouse-core/report/html/report-styles.css]: done - i think?

Comment (patrickhulce) [reply lighthouse-core/report/html/report-styles.css]: wfm!

File: lighthouse-core/report/html/report-styles.css [hunk 2]
```diff
@@ -993,9 +994,17 @@
...
+  background-color: var(--color-amber);
```
Comment (patrickhulce) [inline lighthouse-core/report/html/report-styles.css]: feels like we might want a bit more padding with the new background style 🤷‍♂

**Comment by brendankenny** [comment]
@exterkamp since this is his thing

File: lighthouse-core/report/html/report-styles.css [hunk 3]
```diff
@@ -35,6 +35,7 @@
 .lh-vars {
   /* Palette using Material Design Colors
    * https://www.materialui.co/colors */
+  --color-amber: #FFF8E1; /* 50 */
```
Comment (brendankenny) [inline lighthouse-core/report/html/report-styles.css]: not sure what 50 means

Comment (patrickhulce) [reply lighthouse-core/report/html/report-styles.css]: yeah I was confused too until I looked up the material ui table :) https://www.materialui.co/colors

Comment (brendankenny) [reply lighthouse-core/report/html/report-styles.css]: oh, in that case, should be part of the name. 

> Omit "-{intensity}" if there is only one color of that hue.

isn't simplifying anything if it's helpful enough to have the intensity that we add it in a comment :)

Comment (connorjclark) [reply lighthouse-core/report/html/report-styles.css]: done

File: lighthouse-core/report/html/report-styles.css (hunk 2 above)
Comment (connorjclark) [reply lighthouse-core/report/html/report-styles.css]: done and updated images

File: lighthouse-core/report/html/report-styles.css [hunk 4]
```diff
@@ -993,9 +994,18 @@
 .lh-warnings--toplevel {
   --item-margin: calc(var(--header-line-height) / 4);
   color: var(--report-text-color-secondary);
-  margin: var(--category-padding);
-  padding: var(--category-padding);
+  margin-left: auto;
+  margin-right: auto;
+  width: 50%;
```
Comment (paulirish) [inline lighthouse-core/report/html/report-styles.css]: 50% seems a little arbitrary and has the box edges not matching up with the "grid" of the report.

```cs
max-width: calc(var(--report-width) - var(--category-padding) * 2);
```
this'll have it match most of the report content.

File: lighthouse-core/report/html/report-styles.css [hunk 5]
```diff
@@ -993,9 +994,18 @@
...
+  background-color: var(--color-amber-50);
+  padding: var(--toplevel-warning-padding);
 }
+@media screen and (max-width: 964px) {
```
Comment (paulirish) [inline lighthouse-core/report/html/report-styles.css]: once the max-width is applied you can drop this whole MQ and rule

**Pull request merged**